tableauserverclient/server/request_options.py

```diff
@@ -2,7 +2,7 @@
 
 
 class RequestOptionsBase(object):
-    def apply_query_params(self, url):
```

Comment:
Since this isn't an _ method it is technically "public" -- do we want to preserve it by just having it throw a deprecation warning and then call get_query_params?

Or at a minimum throw a deprecation warning

Reply:
Thanks for pointing that out. I put back the old method, preserving the same behavior with a deprecation warning.